src/common/util.cpp

```diff
@@ -74,6 +75,11 @@ using namespace epee;
 #include <boost/asio.hpp>
 #include <openssl/sha.h>
 
+#ifdef WIN32
```

Comment:
I think we had problems with WIN32 before, and _WIN32 was the "best" one to use.